internal/admission/admission.go

```diff
@@ -91,6 +91,7 @@ func NewGatewayMutationHook(client istioversionedclient.Interface, nsl corev1lis
 }
 
 func (g *GatewayMutationHook) SetupWithManager(mgr manager.Manager) {
+	g.InjectDecoder(admission.NewDecoder(mgr.GetScheme()))
```

Comment:
This is needed or else panic
```
[kanopy-gateway-cert-controller] 2024/12/11 14:33:51 http: panic serving 10.244.0.1:41536: runtime error: invalid memory address or nil pointer dereference
[kanopy-gateway-cert-controller] goroutine 213 [running]:
[kanopy-gateway-cert-controller] net/http.(*conn).serve.func1()
[kanopy-gateway-cert-controller] 	net/http/server.go:1947 +0xb0
[kanopy-gateway-cert-controller] panic({0x17b3d00?, 0x2e29e20?})
[kanopy-gateway-cert-controller] 	runtime/panic.go:785 +0x124
[kanopy-gateway-cert-controller] sigs.k8s.io/controller-runtime/pkg/webhook/admission.(*Decoder).DecodeRaw(0x0?, {{0x4000698c00, 0x5ad, 0x600}, {0x0, 0x0}}, {0x1db9638?, 0x40003c3340})
[kanopy-gateway-cert-controller] 	sigs.k8s.io/controller-runtime@v0.17.2/pkg/webhook/admission/decode.go:77 +0x134
[kanopy-gateway-cert-controller] sigs.k8s.io/controller-runtime/pkg/webhook/admission.(*Decoder).Decode(_, {{{0x400078f980, 0x24}, {{0x4000272e58, 0x13}, {0x40007a9990, 0x7}, {0x40007a9997, 0x7}}, {{0x4000272e70, ...}, ...}, ...}}, ...)
[kanopy-gateway-cert-controller] 	sigs.k8s.io/controller-runtime@v0.17.2/pkg/webhook/admission/decode.go:49 +0x74
[kanopy-gateway-cert-controller] github.com/kanopy-platform/gateway-certificate-controller/internal/admission.(*GatewayMutationHook).Handle(_, {_, _}, {{{0x400078f980, 0x24}, {{0x4000272e58, 0x13}, {0x40007a9990, 0x7}, {0x40007a9997, ...}}, ...}})
[kanopy-gateway-cert-controller] 	github.com/kanopy-platform/gateway-certificate-controller/internal/admission/admission.go:102 +0xb4
```

I'm not sure why it was working before

Reply:
I believe this was automatic before.